iphone/Classes/MediaModule.m

```diff
@@ -624,6 +624,14 @@ -(id)hasAudioPermissions:(id)unused
 #ifdef USE_TI_MEDIAREQUESTAUDIOPERMISSIONS
 -(void)requestAudioPermissions:(id)args
 {
+    DEPRECATED_REPLACED(@"Media.requestAudioPermissions", @"6.1.0", @"Media.requestRecorderPermission");
+    [self requestRecorderPermission:args];
+}
+#endif
+
+#ifdef USE_TI_MEDIAREQUESTRECORDPERMISSIONS
+-(void)requestRecorderPermission:(id)args
```

Comment:
Don't forget to align the macro to the suggested method as well. And in this case, this won't work on the device either, because `USE_TI_MEDIAREQUESTRECORDPERMISSIONS` won't be generated when using `Ti.Media. requestRecorderPermission` (see the `s` at the end).